README.md

```diff
@@ -32,6 +32,20 @@ Percy.logger.error { 'error message' }
 
 Prefer the block form usage `Percy.logger.debug { 'message' }` over `Percy.logger.debug('message')` because it is slightly more efficient when the log will be excluded by the current logging level. For example, if the log level is currently `info`, then a `debug` log in block form will never evaluate or allocate the message string itself.
 
+## Percy::ProcessHelpers
+
+### `gracefully_kill(pid[, grace_period_seconds: 10])`
+
+Returns `true` if the process was successfully killed, or `false` if the process did not exist or its exit status was already collected.
```

Comment:
Interestingly, the tests to exercise this case were pretty simple — just `Process.wait` on the pid to collect the status first, then call our gracefully_kill method: https://github.com/percy/percy-common/pull/2/files#diff-3695a3e2e701faaee28df9efe7532700R22